src/junit_reporter.js

```diff
@@ -31,13 +31,17 @@
             pad(d.getMinutes()) + ':' +
             pad(d.getSeconds());
     }
+    function escapeControlChars(str) {
+        // Remove control character from Jasmine default output
+        return str.replace(/[\x1b]/g, "");
+    }
     function escapeInvalidXmlChars(str) {
-        return str.replace(/\&/g, "&amp;")
+        const escaped = str.replace(/\&/g, "&amp;")
```

Comment:
This must not use `const` in any of the reporter source files, as it may be used on a browser version that does not support `const`—this should be `var` for maximum compatibility. New keywords are fine to use in the spec files (as they only run in node), but reporter files need to be much more backwards compatible. 

I am happy to merge this PR if it is updated to use `var`.

Thanks.